plugins/maven-dependency-resolver/build.gradle

```diff
@@ -3,13 +3,57 @@ import org.robolectric.gradle.RoboJavaModulePlugin
 
 apply plugin: RoboJavaModulePlugin
 apply plugin: DeployedRoboJavaModulePlugin
+apply plugin: 'kotlin'
+apply plugin: "com.diffplug.spotless"
+
+spotless {
+    kotlin {
+        target '**/*.kt'
+        ktfmt('0.42').googleStyle()
+    }
+}
+
+tasks.withType(GenerateModuleMetadata) {
+    // We don't want to release gradle module metadata now to avoid
+    // potential compatibility problems.
+    enabled = false
+}
+
+compileKotlin {
+    // Use java/main classes directory to replace default kotlin/main to
+    // avoid d8 error when dexing & desugaring kotlin classes with non-exist
+    // kotlin/main directory because utils module doesn't have kotlin code
+    // in production. If utils module starts to add Kotlin code in main source
+    // set, we can remove this destinationDirectory modification.
+    destinationDirectory = file("${projectDir}/build/classes/java/main")
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
```

Comment:
```suggestion
    compilerOptions.jvmTarget = org.jetbrains.kotlin.gradle.dsl.JvmTarget.JVM_1_8
```

Reply:
Looks like it is better. Thanks, I will update it later.

Reply:
Cause `kotlinOptions.jvmTarget` is deprecated from Kotlin 1.8